R/csv-utils.R

```diff
@@ -1,19 +1,42 @@
-#' @keywords internal
-mrgda_read_csv <- function(.file) {
+#' Read csv file
+#'
+#' @description
+#' Reads csv files and formats the data to replace any "." with
+#' NA values.
+#'
+#' @param .file file path to csv file
+#'
+#' @export
+read_nm_csv <- function(.file) {
 
-  data.table::fread(
-    file = .file,
-    sep = ",",
-    quote = FALSE,
-    na.strings = "."
-  ) %>%
+  df <-
+    data.table::fread(
+      file = .file,
+      sep = ",",
+      quote = FALSE,
+      na.strings = "."
+    ) %>%
     as.data.frame() %>%
```

Comment:
Rather than force coercion out of data.table, include an argument called `data.table` and set the default to `FALSE`; then people can opt in to keep working on a data.table.